core_memory_recall.py

```
@@ -0,0 +1,194 @@
+                        profile_tags = [t for t in profile_tags if relevance(t) > 0]
+                    for tag in profile_tags[:5]:
+                        display_memories.append(MemoryItem(
+                            id=f"profile_{tag['type']}",
+                            memory_type="user_profile",
+                            judgment=tag["value"],
+                            reasoning=f"[画像] {tag['type']} | priority: self_reported",  # 标记优先级（行为数据覆盖）
+                            tags=[tag["type"], "profile"],
+                            strength=10,
+                            created_at=0.0,
+                        ))
+                except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)